blocks/core/ff_module/ff_module-task-event-status/ff_module-task-event-status.less

```diff
@@ -2,20 +2,17 @@
 
     display: inline-block;
     font-size: .875rem;
-    padding: @ff_size_spacing_min*0.25 @ff_size_spacing_min;
+
     // conceptually if not actually a child
     &__transition-wrapper {
         position: relative;
     }
 
     &__icon {
-        margin: -.4em .4em -.4em -.25em;
+        margin-right: 0.25rem;
```

Comment:
It's layout but it's pretty fundamental and most themes won't have to override this, in which case I don't think it's necessary to add it to every theme when it can appear here.